soh/soh/SohMenuBar.cpp

```diff
@@ -962,6 +962,10 @@ void DrawEnhancementsMenu() {
             UIWidgets::Tooltip("Always shows dungeon entrance icons on the minimap");
             UIWidgets::PaddedEnhancementCheckbox("Show Gauntlets in First Person", "gFPSGauntlets", true, false);
             UIWidgets::Tooltip("Renders Gauntlets when using the Bow and Hookshot like in OOT3D");
+            if (UIWidgets::PaddedEnhancementCheckbox("Colour Temple of Time's Medallions", "gToTMedallionsColours", true, false)) {
+                PatchToTMedallions();
+            }
+            UIWidgets::Tooltip("Colours (or not) the Temple of Time Medallions");
```

Comment:
This tooltip is basically the same as the menu item name itself. Maybe a description like this would be better:
```suggestion
            UIWidgets::Tooltip("When medallions are collected, the medallion imprints around the Master Sword pedestal in the Temple of Time will become colored");
```